lib/redix/set.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Redix.Set do
+  if Version.compare(System.version, "1.1.0") == :lt do
```

Comment:
Sound a good solution to drop OTP 17 & Elixir < 1.2.

That's what the guys of connection did, so we'll be coherent.

I have updated the PR to require the elixir version of 1.2 and removed the shim.

Thanks again !